packages/manager/modules/pci/src/projects/project/workflow/add/add.module.js

```diff
@@ -0,0 +1,35 @@
+import angular from 'angular';
+
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import '@ovh-ux/ng-ovh-swimming-poll';
+import 'ovh-api-services';
+import 'ovh-ui-angular';
+
+import component from './add.component';
+import routing from './add.routing';
+import workflowType from './type';
+import resources from './resources';
+import schedule from './schedule';
+import generalInfo from './general';
+import PciProjectsProjectInstanceService from '../../instances/instances.service';
+import PciProjectRegions from '../../instances/add/regions-list/regions-list.service';
+
+const moduleName = 'ovhManagerPciProjectWorkflowAddModule';
+
+angular.module(moduleName, [
+  workflowType,
+  resources,
+  schedule,
+  generalInfo,
+  'ngOvhCloudUniverseComponents',
+  'ngOvhSwimmingPoll',
+  'oui',
+  'ovh-api-services',
+])
+  .config(routing)
+  .component('ovhManagerPciProjectWorkflowAdd', component)
+  .service('PciProjectRegions', PciProjectRegions)
```

Comment:
⚠️ This will cause conflicts with https://github.com/ovh-ux/manager/pull/961

Reply:
Translations seems not loaded.

```suggestion
  .service('PciProjectRegions', PciProjectRegions)
  .run(/* @ngTranslationsInject:json ./translations */)
```

Reply:
I did not see any error message on this, I verified and it looked like all required translations are loaded.

Reply:
@radireddy Could you please addressed @marie-j's feedback?

Reply:
I removed it, its not required anymore after latest changes.